exercises/split-second-stopwatch/metadata.toml

```
@@ -0,0 +1,4 @@
+title = "Split-second Stopwatch"
```

```suggestion
title = "Split-Second Stopwatch"
```

Match case to the intro